src/pages/_document.tsx

```diff
@@ -22,7 +23,27 @@ export default class MyDocument extends Document<MyDocumentProps> {
     return (
       <html>
         <Head>
-          <title>My page</title>
+          <meta httpEquiv="Content-Type" content="text/html" charSet="utf-8" />
+          <meta httpEquiv="Content-Security-Policy" content={Csp} />
+          <meta name="referrer" content={Referrer} />
+          <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1, shrink-to-fit=no"
+          />
+          <meta
+            name="author"
+            content="back-end= Renan Lima e Rodrigo Gonçalves, design= Carla Robles, front-end= Matheus Schiffer"
+          />
+          <meta name="description" content="Associacao Share Website" />
+          <meta name="application-name" content="Share Frontpage" />
+          <meta name="keywords" content="SEO" />
+          <meta httpEquiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+          <title>Associacao Share</title>
```

Comment:
title não deve ser colocado em _document, Se for um title global, deve ir em _app